tests/components/todoist/test_calendar.py

```diff
@@ -216,9 +216,6 @@ async def test_calendar_custom_project_unique_id(
     entity = entity_registry.async_get("calendar.all_projects")
     assert entity is None
 
-    state = hass.states.get("calendar.all_projects")
-    assert state.state == "off"
-
```

Comment:
This assertion was actually wrong.  Because the initial state wasn't being updated when the entity was added this assertion was passing before this change.  I decided to remove this particular assertion since it's not really related to this test.